.eslintrc.js

```diff
@@ -0,0 +1,57 @@
+module.exports = {
+	env: {
+		browser: true,
+		es2021: true,
+	},
+	parserOptions: {
+		ecmaVersion: '2021',
+	},
+	globals: {
+		App: 'readonly',
+		jQuery: 'readonly',
+		_: 'readonly',
+		$: 'readonly',
+	},
+	overrides: [
+		{
+			files: [
+				'web/assets/js/fx/*.js',
+				'web/assets/js/modules/*.js',
+				'web/assets/js/utils/*.js',
+				'web/assets/js/com/*.js',
+			],
+			rules: {
+				'no-bitwise': 'error',
+				'no-undef': 'error',
+				'no-global-assign': 'error',
+				'no-console': ['error', { allow: ['warn', 'error'] }],
+				curly: 'error',
+				eqeqeq: 'error',
+				'guard-for-in': 'error',
+				'no-extend-native': 'error',
+				'no-unused-vars': 'error',
+				'no-use-before-define': 'error',
+				'new-cap': 'error',
+				'no-empty': 'error',
+				strict: ['error', 'function'],
+				quotes: ['error', 'single'],
+				'max-params': ['error', 5],
+				'max-len': [
```

Comment:
ignoreComments, ignoreTrailingComments, ignoreTemplateLiterals, ignoreStrings should be `false` the objective is to avoid hiding code on an horizontal scroll since people are not inclined to consult the documentation and implementation hidden by this scroll.

Reply:
Good point for the comments, but sometimes we have long strings or template literal (ie, html inside a string), and with all the prior indentation it can get really hard to fit in 100 characters.

Reply:
I get that and in those cases we need to concatenate multiple strings. We should never hide stuff.